doc/ref_model/chapters/chapter07.rst

```diff
@@ -394,52 +394,116 @@ ZTA principles applied to Cloud infrastructure components are the following:
 
 Zero Trust principles should also be applied to cloud-native applications. With the increasing use of these applications which are designed with microservices and deployed using containers as packaging and Kubernetes as an orchestrator, the security of east-west communications between components must be carefully addressed. The use of secured communication protocols brings a first level of security, but considering each component as non-trustworthy will minimize the risk for applications to be compromised. A good practice is to implement the proxy-based service mesh which will provide a framework to build a secured environment for microservices-based applications, offering services such as service discovery, authentication and authorisation policies enforcement, network resilience, and security monitoring capabilities. The two documents, `NIST SP 800-204A <https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-204A.pdf>`__\ (Building Secure Microservices-based Applications Using Service-Mesh Architecture) and `NIST SP 800-204B <https://csrc.nist.gov/publications/detail/sp/800-204b/final>`__\ (Attribute-based Access Control for Microservices-based Applications Using a Service Mesh), describe service mesh, and provide guidance for service mesh components deployment.
 
-Open Source Software Security
------------------------------
+Software Supply Chain Security
+------------------------------
+
+Software supply chain attacks are increasing worldwilde and can cause serious damages. Many entreprises and 
+organisations are experiencing these threats. Software suppliers can be targeted and by rebound effect their 
```

Comment:
"rebound"?
```suggestion
organisations are experiencing these threats. Software suppliers can be targeted, and by rebound affect their 
```

Reply:
@pgoyal01 in this sentence "Rebound effect" can not be replaced by affect. I can change to : and by rebound their customers can be affected. it would works?

Reply:
Text changed, please Check if Okay